sparse/usr/share/ngfd/plugins.d/60-droid-vibrator-device.ini

```diff
@@ -0,0 +1,36 @@
+[droid-vibrator]
```

Comment:
If this file is identical to the one from ngfd-plugin-droid-vibrator then no need to copy here.

Reply:
It's not. It has different values. The one in ngfd-plugin-droid-vibrator uses [values from the old qtfeedback plugin](https://github.com/mer-hybris/qt5-feedback-haptics-droid-vibrator/blob/master/droid-vibrator-device.ini.example). I don't know if those values are used somewhere in practise or if we should make these the default.

Reply:
ok, good :)